modules/storages/app/controllers/storages/admin/project_storages_controller.rb

```diff
@@ -127,9 +128,7 @@ def destroy
   end
 
   def destroy_info
-    @project_storage_to_destroy = @object
-
-    render "/storages/project_settings/destroy_info"
```

Comment:
As far as I can tell, the `destroy_info` view is used nowhere else, so I guess the file can be removed as well.